libs/openFrameworks/math/ofMatrix4x4.cpp

```diff
@@ -931,10 +931,10 @@ void ofMatrix4x4::makeLookAtMatrix(const ofVec3f& eye,const ofVec3f& center,cons
 	ofVec3f xaxis = up.getCrossed(zaxis).getNormalized();
 	ofVec3f yaxis = zaxis.getCrossed(xaxis);
 
-	_mat[0].set(xaxis.x, xaxis.y, xaxis.z, 0);
-	_mat[1].set(yaxis.x, yaxis.y, yaxis.z, 0);
-	_mat[2].set(zaxis.x, zaxis.y, zaxis.z, 0);
-	_mat[3] = eye;
```

Comment:
That's a great catch! An `ofVec3f` implicitly cast to `ofVec4f` will have `.w` set to 0, which will lead to invalid homogenous coordinates.